frontend/src/components/homepage/Filter.tsx

```diff
@@ -79,9 +76,11 @@ const Filter = ({
   );
 
   const maxApprovedLevel = approvedLanguages[language];
-  const levelOptions = Array.from(Array(maxApprovedLevel).keys()).map(
-    (val) => `Level ${val + 1}`,
-  );
+  const levelOptions = isDisabled
+    ? ["Level 1", "Level 2", "Level 3", "Level 4"]
+    : [...Array(maxApprovedLevel + 1).keys()]
```

Comment:
nit: add quick comment here that the else of the ternary is building a `Level 1 ... Level max` array